innovation_sweet_spots/analysis/prototyping/public_discourse_analysis/Example_data_collection.py

```diff
@@ -0,0 +1,141 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.3
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Collect Guardian articles, subset relevant documents and extract text
+#
+# - Fetch articles using Guardian API
+# - Extract metadata and text from html
+
+# %% [markdown]
+# ## 1. Import dependencies
+
+# %%
+import os
+import pandas as pd
+import csv
+import json
+from collections import defaultdict
+import pickle
+
+# %%
+# Change first element to location of project folder.
+os.chdir(os.path.join('/Users/jdjumalieva/Documents/Analysis/', 'innovation_sweet_spots'))
```

Comment:
@djyldyz I think George meant whether the `innovation_sweet_spots` module is installed in a way that you can call `import innovation_sweet_spots` from any folder in the project. It looks it is not the case, as you need to manually specify the working directory to be the repo root. You can additionally test this by navigating in the terminal into another directory besides the repo root directory, running ipython and trying `import innovation_sweet_spots`

So, I think my repo installation instructions haven't been clear. Could you try running the following in the terminal, from the repo root directory?
```
pip install -e .
```
This should install innovation_sweet_spots, so that you can call it from anywhere.

Reply:
By the way (you probably already know this) - once this is working, you can get the full path to local repo directory by simply calling
`from innovation_sweet_spots import PROJECT_DIR`